lib/fs.js

```diff
@@ -132,7 +132,7 @@ function makeCallback(cb) {
   }
 
   return function() {
-    return cb.apply(null, arguments);
+    return cb(...arguments);
```

Comment:
There are a few other combinations that should be benchmarked:
1. `cb.bind(undefined)`
2. `function (...args) { cb(...args) }`
3. `(...args) => cb(...args)`

I'll try to check some.